gcloud/bigquery/table.py

```diff
@@ -22,6 +22,35 @@
 from gcloud.bigquery._helpers import _prop_from_datetime
 
 
+class SchemaField(object):
+    """Describe a single field within a table schema.
+
+    :type name: string
+    :param name: the name of the field
+
+    :type field_type: string
+    :param field_type: the type of the field (one of 'STRING', 'INTEGER',
+                       'FLOAT', 'BOOLEAN', 'TIMESTAMP' or 'RECORD')
```

Comment:
Opened https://github.com/GoogleCloudPlatform/gcloud-python/issues/1031 to track this issue (which affects more places than this).